app/controllers/organization_controller.rb

```diff
@@ -1,19 +1,15 @@
 class InseeProxyController < ApplicationController
   # GET /insee-proxy/naf/1
   def naf
-    response = HTTP
-      .headers(accept: "application/json")
-      .post(
-        "https://www.insee.fr/fr/metadonnees/nafr2/consultation",
-        json: {q: params[:id]}
-      )
+    response = File.read("./public/codes_naf.json")
+    parsed_response = JSON.parse(response)
 
-    parsed_response = response.parse
+    code = params.fetch(:id, "").gsub('.', '')
 
-    if parsed_response["documents"].nil? || parsed_response["documents"][0].nil?
-      render status: :not_found, json: {}
+    if(parsed_response[code].nil?)
+      render status: :ok, json: { libelle: "Code inconnu" }
```

Comment:
Rather than `libelle`, I'd rather use `message` for consistency purpose. See: https://github.com/betagouv/signup-back/blob/29c5c4c0070050c0be070523bff0177ea56caf6e/app/controllers/application_controller.rb#L26-L73